webpack.config.js

```diff
@@ -91,6 +94,11 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 } else {
 	webpackConfig.entry.vendor = [ 'react', 'store', 'page', 'wpcom', 'jed', 'debug' ];
 	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( 'vendor', '[name].[hash].js' ) );
+	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( {
```

Comment:
Should we give the chunk a useful `name` ? I'll likely not be the only one thrown off by `0.9081fbc2a41d0d5395c0.js` 😄